crates/oxc_linter/src/snapshots/require_await.snap

```diff
@@ -0,0 +1,63 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: require_await
+---
+  ⚠ eslint(require-await): Async function has no 'await' expression.
+   ╭─[require_await.tsx:1:1]
+ 1 │ async function foo() { doSomething() }
```

Comment:
Would you like shrink it function name `foo`?

https://oxc-project.github.io/docs/contribute/linter.html#pin-point-the-error-message-to-the-shortest-code-span

Reply:
done